modules/raft/src/main/java/org/apache/ignite/internal/raft/Loza.java

```diff
@@ -314,25 +325,29 @@ public <T extends RaftGroupService> CompletableFuture<T> startRaftGroupNodeAndWa
             RaftGroupListener lsnr,
             RaftGroupEventsListener eventsLsnr,
             RaftNodeDisruptorConfiguration disruptorConfiguration,
-            @Nullable RaftServiceFactory<T> factory
+            @Nullable RaftServiceFactory<T> factory,
+            RaftOptionsConfigurator storageConfigurator
     ) throws NodeStoppingException {
         if (!busyLock.enterBusy()) {
             throw new NodeStoppingException();
         }
 
         try {
-            CompletableFuture<T> startRaftServiceFuture = startRaftGroupNodeInternal(
+            RaftGroupOptions raftGroupOptions = RaftGroupOptions.defaults();
+
+            storageConfigurator.configure(raftGroupOptions);
+
+            raftGroupOptions.ownFsmCallerExecutorDisruptorConfig(disruptorConfiguration);
```

Comment:
Why don't we reduce this method from Loza and add this setter to the raft option configuration closure (msRaftConfigurator)?